tests/test_label_space.py

```diff
@@ -0,0 +1,11 @@
+import ansys.dpf.core as dpf
+
+
+def test_label_space_(server_type):
+    reference = {"test": 1, "various": 2}
+    ls = dpf.LabelSpace(label_space=reference)
+    assert dict(ls) == reference
+    assert str(ls)
+    reference = {"test": 1, "various": 2}
+    ls.fill(label_space=reference)
+    print(ls)
```

Comment:
Maybe you should not leave a `print` in the test but test the changes that you did right above
```suggestion
```

Reply:
Yes sorry that's a rogue forgotten print